ci/closures.sh

```diff
@@ -1,12 +1,17 @@
 #!/bin/bash
-set -xe
-set -o pipefail
+set -x
 
 # Enables printing of all potential GC allocation sources to stdout
 export DFLAGS=-vgc
 
 # Prepare sources
-beaver dlang make d2conv
+beaver dlang make d2conv || exit 1
 
-# Ensure that there are no lines about closure allocations in the output
-! beaver dlang make fasttest 2>&1 | grep "closure"
+# Run tests and write compiler output to temporary file
+compiler_output=`mktemp`
```

Comment:
Minor note: we could use something like `mktemp --tmpdir closure-check.XXXXXXXX` to generate a temp file whose name clearly associates it with this test.

Reply:
What for? It is printed to stdout for inspection and we don't preserve those tmp files.